java/client/src/main/java/glide/api/commands/SetCommands.java

```diff
@@ -0,0 +1,89 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.commands;
+
+import java.util.Set;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Set Commands interface.
+ *
+ * @see <a href="https://redis.io/commands/?group=set">Set Commands</a>
+ */
+public interface SetCommands {
+    /**
+     * Add specified members to the set stored at <code>key</code>. Specified members that are already
+     * a member of this set are ignored.
+     *
+     * @see <a href="https://redis.io/commands/sadd/">redis.io</a> for details.
+     * @param key The <code>key</code> where members will be added to its set.
+     * @param members A list of members to add to the set stored at <code>key</code>.
+     * @return The number of members that were added to the set, excluding members already present.
+     * @remarks
+     *     <ul>
+     *       <li>If <code>key</code> does not exist, a new set is created before adding <code>members
+     *           </code>.
+     *       <li>If <code>key</code> holds a value that is not a set, an error is returned.
```

Comment:
We have decided to remove from all commands docs the message: `If key holds a value that is not a x , an error is returned.`